public/app/plugins/datasource/cloudwatch/tracking.ts

```diff
@@ -0,0 +1,93 @@
+import { DashboardLoadedEvent } from '@grafana/data';
+import { reportInteraction } from '@grafana/runtime';
+
+import { isCloudWatchLogsQuery, isCloudWatchMetricsQuery, isCloudWatchQuery } from './guards';
+import pluginJson from './plugin.json';
+import { CloudWatchMetricsQuery, MetricEditorMode, MetricQueryType } from './types';
+
+interface CloudWatchOnDashboardLoadedTrackingEvent {
+  grafana_version?: string;
```

Comment:
With the event-bus solution, we now have access to the plugin version, which might be useful to add ds_version: pluginJson.info.version ?

Reply:
For core plugins, the ds_version will always be the same as grafana version so it's redundant. But should be very interesting for external plugins!

Reply:
oh yes of course 🤦‍♀️ 😄 !